packages/plugin-commands-installation/src/installDeps.ts

```diff
@@ -222,6 +222,14 @@ when running add/update with the --workspace option')
       params = createWorkspaceSpecs(params, workspacePackages)
     }
   }
+
+  if (opts.update) {
+    params = params.filter((param) => {
+      const packageName = param.slice(0, param.lastIndexOf('@'))
+      return !(manifest?.pnpm?.update?.ignore ?? []).includes(packageName)
+    })
+  }
```

Comment:
this doesn't makes sense to me. So if I run `pnpm update express` and express is in the ignore list then express will not be updated? I think the ignore list should only ignore updates on `pnpm update`. On named updates either all the listed dependencies should be updated or an error should be thrown.

Reply:
Oh, you are right. it is not necessary to ignore packages specified with `pnpm update`'s args. I will fix this 🙏